index.js

```diff
@@ -205,7 +210,9 @@ program
 
 program.parse(process.argv);
 
-if (!program.args.length || !ACCEPTED_COMMANDS.includes(process.argv[2])) {
+if (!program.args.length
+  || !ACCEPTED_COMMANDS.includes(process.argv[2])
+  || !ACCEPTED_COMMANDS_ALIAS.includes(process.argv[2])) {
```

Comment:
The warning now shows for all commands, even the non-alias versions. The logic should be to check that it does not exist in both the non-alias **and** the alias list.